npm-run/action.yml

```diff
@@ -25,18 +25,12 @@ inputs:
 runs:
   using: composite
   steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
       with:
         fetch-depth: 0
         ref: ${{ inputs.ref }}
-    - name: Cache node_modules
-      id: cache-npm
-      uses: actions/cache@v3
-      with:
-        key: ${{ hashFiles( format('{0}/{1}', inputs.directory, 'package.json') ) }}
-        path: ${{ inputs.directory }}/node_modules
     - name: Set up Node
-      uses: actions/setup-node@v2
+      uses: actions/setup-node@v3
```

Comment:
@andreyudentsovottofeller please add cache support like described in docs https://github.com/actions/setup-node#caching-global-packages-data

Reply:
@flyin 
Added a change